packages/asc-ui/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { render, cleanup, fireEvent } from '@testing-library/react'
+import { ascDefaultTheme, ThemeProvider } from '@datapunt/asc-core'
+import Checkbox from './Checkbox'
+
+const theme = {
+  ...ascDefaultTheme,
+}
+
+describe('Checkbox', () => {
+  let container: any
+  const onChangeMock = jest.fn()
+
+  beforeEach(() => {
+    cleanup()
+    ;({ container } = render(
```

Comment:
Is this prettier?

Reply:
Yes it is, but needed. The semi is necessary because otherwise it wont even compile. Imagine if the code would be minified, you'll het something like this `cleanup()({ container... })` it would try to invoke a method returned by cleanup, which doesn't make sense in this case